meta-ostro/classes/ostro-image.bbclass

```diff
@@ -280,6 +280,10 @@ NOHDD = "1"
 # the rootfs. WARNING: any change to this value will trigger a
 # rebuild (and re-sign, if enabled) of the combo EFI application.
 ROOTFS_PARTUUID_VALUE = "12345678-9abc-def0-0fed-cba987654321"
+# The second value is needed for the system installed onto
+# the device's internal storage in order to mount correct rootfs
+# when an installation media is still inserted into the device.
+INT_STORAGE_ROOTFS_PARTUUID_VALUE = "12345678-9abc-def0-0fed-cba987654320"
```

Comment:
I promise this is the last nitpicking: how about renaming ROOTFS_PARTUUID_VALUE to something like REMOVABLE_MEDIA_ROOTFS_PARTUUID_VALUE ?

Because that's going to be its sole use, right?

Side comment: I would have preferred to see also the installer, so that the entire use case could be reviewed.
However, for now, apart from the ROOTFS_PARTUUID_VALUE renaming, lgtm.